src/Compilers/Core/Portable/CodeAnalysisResources.resx

```diff
@@ -785,4 +788,7 @@
   <data name="SigningTempPathUnavailable" xml:space="preserve">
     <value>The temporary path for legacy file signing is unavailable.</value>
   </data>
+  <data name="Type_0_does_not_have_expected_constructor" xml:space="preserve">
```

Comment:
>Type_0_does_not_have_expected_constructor [](http://example.com/codeflow?start=14&length=41)

Consider matching the naming style used for these resources: `Type0DoesNotHaveExpectedConstructor`